packages/flutter/lib/src/material/snack_bar_theme.dart

```diff
@@ -77,6 +78,11 @@ class SnackBarThemeData extends Diagnosticable {
   /// otherwise.
   final Color disabledActionTextColor;
 
+  /// Used to configure the [DefaultTextStyle] for the [SnackBar.content] widget.
+  ///
+  /// If null, [SnackBar] defaults to [ThemeData.textTheme.subhead].
```

Comment:
This isn't necessarily true right? It specifically defaults to darkTheme's `textTheme.subhead`

Reply:
Adjusted the wording